features/breadcrumbs.feature

```diff
@@ -0,0 +1,9 @@
+Feature: Breadcrumbs
+  In order to use the skip link,
+  As a user
+  I want the breadcrumbs to be outside the main content
+
+  Scenario: When there are breadcrumbs on the page
+    Given the page exists
+    When I visit the page
```

Comment:
When I visit /search

Reply:
Or instead of doing all of that you can simply reuse an existing step, please, see a comment above.